### PR TITLE
Restructure the Queries Get Started Guide

### PR DESCRIPTION
**Pull request by praneesha**
## Purpose
Restructure the queries get started guide.
> Fixes #

## Checklist

- [ ] **Page addition**
  - [ ] Add `permalink` to pages.

- [ ] **Page removal**
  - [ ] Remove entry from corresponding left nav YAML file.
  - [ ] Add `redirect_from` on the alternative page.
  - [ ] If no alternative page, add redirection on the `redirections.js ` file.

- [ ] **Page rename**
  - [ ] Add front-matter `redirect_from`.
  - [ ] Add front-matter `redirect_to:` (if applicable).

- [ ] **Page restrcuture**
  - [ ] Add `permalink` to pages.
  - [ ] Add front-matter `redirect_from`.
  - [ ] Add front-matter `redirect_to:` (if applicable).

### CODE REVIEW

File: learn/get-started/work-with-data-using-queries-in-ballerina.md [hunk 1]
```diff
@@ -18,42 +18,44 @@ redirect_from:
 
 Ballerina has first-class support for writing SQL-like queries to process data. Language-integrated queries can process any Ballerina iterable.
 
-In this tutorial, you will be writing queries to filter, sort, and join with different data sets and produce new data sets.
-
 ## Set up the prerequisites
 
-To complete this tutorial, you need:
+To run this guide, you need the following prerequisites:
```
Comment (gimantha) [inline learn/get-started/work-with-data-using-queries-in-ballerina.md:23]: Is it correct to say run here? How about follow or similar wording?

File: learn/get-started/work-with-data-using-queries-in-ballerina.md [hunk 2]
```diff
@@ -121,23 +165,60 @@ public function findCountriesByHighestNoOfDeaths(table<CovidEntry> dataTable, in
...
+}
+
+public function findCountriesByHighestNoOfDeaths(table<CovidEntry> dataTable, int n) returns [string, decimal][] {
+    [string, decimal][] countriesWithDeaths = from CovidEntry entry in dataTable
+        order by entry.deaths descending
+        limit n
+        select [entry.country, entry.deaths];
+    return countriesWithDeaths;
+}
+
```
Comment (gimantha) [inline learn/get-started/work-with-data-using-queries-in-ballerina.md:204]: This is the only part related to sorting. Can we highlight this part only and say, below is the complete code after adding the sorting logic which is highlighted.?

File: learn/get-started/work-with-data-using-queries-in-ballerina.md [hunk 3]
```diff
@@ -161,14 +244,51 @@ public function findRecoveredPatientsOfCountries(table<CovidEntry> dataTable, st
...
+public function findRecoveredPatientsOfCountries(table<CovidEntry> dataTable, string[] countries) returns [string, decimal][] {
+    [string, decimal][] countriesWithRecovered = from CovidEntry entry in dataTable
+        join string country in countries on entry.country equals country
+        select [entry.country, entry.recovered];
+    return countriesWithRecovered;
+}
```
Comment (gimantha) [inline learn/get-started/work-with-data-using-queries-in-ballerina.md:290]: Commen in https://github.com/ballerina-platform/ballerina-dev-website/pull/4596/files#r917536655 applies to here also

File: learn/get-started/work-with-data-using-queries-in-ballerina.md [hunk 4]
```diff
@@ -214,13 +344,60 @@ public function printErroneousData(table<CovidEntry> dataTable) {
...
+
+To call the `findRecoveredPatientsOfCountries` function at the end to get the number of recovered patients, add the code below to the `main` function of the `main.bal` file. 
 
 ```ballerina
+printErroneousData(covidTable);
+```
+
+## The completed code for finding
+
+Below is the complete code for the finding function.
```
Comment (gimantha) [inline learn/get-started/work-with-data-using-queries-in-ballerina.md:364]:
```suggestion
Below is the complete code after adding the finding function.
```
We can also say like that and highlight the newly added logic. wdyt?

File: learn/get-started/work-with-data-using-queries-in-ballerina.md (hunk 1 above)
Comment (praneesha) [reply learn/get-started/work-with-data-using-queries-in-ballerina.md:23]: @dakshitha - WDYT?

File: learn/get-started/work-with-data-using-queries-in-ballerina.md (hunk 2 above)
Comment (praneesha) [reply learn/get-started/work-with-data-using-queries-in-ballerina.md:204]: Fixed.

File: learn/get-started/work-with-data-using-queries-in-ballerina.md (hunk 4 above)
Comment (praneesha) [reply learn/get-started/work-with-data-using-queries-in-ballerina.md:364]: Fixed.

**Review by dakshitha — APPROVED** [review]

File: learn/get-started/work-with-data-using-queries-in-ballerina.md (hunk 1 above)
Comment (praneesha) [reply learn/get-started/work-with-data-using-queries-in-ballerina.md:23]: Will be fixed via https://github.com/ballerina-platform/ballerina-dev-website/pull/4696.

**Review by gimantha — APPROVED** [review]

**Pull request merged**